app/over_react_redux/todo_client/.gitignore

```diff
@@ -0,0 +1,23 @@
+.packages
+.pub
+packages
+*.over_react.g.dart
```

Comment:
I think you're thinking about the `over_react` package @greglittlefield-wf  - this app is its own package - which does not specify `build_to: source`.

Reply:
Whoops, sorry, I meant `build_to: cache`. I always get those mixed up 😓 

But yeah, since they're `cache`, they should only ever exist inside the `.dart_tool` directory within this repo.